src/meta_utils.py

```diff
@@ -0,0 +1,528 @@
+import numpy as np
+import pandas as pd
+from typing import Optional, Tuple, Callable
+
+# (1) Crop land **mapping** <- MOST GENERAL
+#     -> NOTE: With crop land map there is no 'final' agreement between two labeler 
+#              sets b/c there is typically no *forced* agreement or resolvement.
+
+# (2) Crop land **area estimation**
+#     -> NOTE: With area estimation there *is* final agreement between the two labeler sets. <- MOST COMMON
+#     -> NOTE: Additionally; area estimation may also be for either single year (map) or 
+#              multi-year (area change).
+
+# (3) With area estimation there are additionally two types:
+#     -> Single-year crop map area estimation
+#     -> Multi-year crop map change area estimation 
+
+# (3) Difference in **mapping** and **area estimation**
+#     -> mapping : two csv files (set 1, set 2)
+#     -> area est. : three csv files (set 1, set 2, 'final')
+
+# (4) Goal:
+#     -> Generalize functions st behavior adjusted depending on if labeling project is **mapping** 
+#        or **area estimation** 
+#     -> Don't require additional script file; instead have two separate notebooks for mapping
+#        and area estimation but all util functions in one .py
```

Comment:
This is neatly written! Typically when I see a comment in a file it usually explains the purpose of the code in the file. This comment seems more like context or a note and I feel it may be more fitting in the Github Wiki (and can be linked here) or should be associated with functions (code). Let me know what you think!